src/coreclr/tools/Common/CommandLineHelpers.cs

```diff
@@ -177,16 +207,15 @@ public static void MakeReproPackage(string makeReproPath, string outputFilePath,
                 Dictionary<string, string> outputToReproPackageFileName = new();
 
                 List<string> rspFile = new List<string>();
-                foreach (var option in res.CommandResult.Command.Options)
+                foreach (CliOption<object> option in res.CommandResult.Command.Options)
                 {
-                    if (!res.HasOption(option) || option.Name == "make-repro-path")
+                    if (res.FindResultFor(option) == null || option.Name == "make-repro-path")
                     {
                         continue;
                     }
 
-                    IValueDescriptor descriptor = option;
                     object val = res.CommandResult.GetValue(option);
-                    if (val is not null && !(descriptor.HasDefaultValue && descriptor.GetDefaultValue().Equals(val)))
+                    if (val is not null)
```

Comment:
@adamsitnik, this was the functionality compromise. I couldn't find a way to obtain the default value of `CliOption`.

Reply:
@am11 thank you for your feedback! I've opened https://github.com/dotnet/command-line-api/pull/2218 to address that.

Reply:
Do we need a new release with that change? From latest build: https://dev.azure.com/dnceng-public/public/_build/results?buildId=303826&view=results